packages/elements/src/card/index.ts

```diff
@@ -147,8 +159,10 @@ export class Card extends BasicElement {
    * @returns {void}
    */
   private closeMenu (): void {
-    if (this.menuElement) {
+    if (this.menuElement && this.menuElement.opened) {
       this.menuElement.opened = false;
+      this.openMenuElement?.setAttribute('aria-expanded', 'false');
+      this.openMenuElement?.focus(); // Need refocus on button because A11Y JAWS lost focus from the element
```

Comment:
Interesting, this is the default behaviour of `overlay` to re-focus back on the element that has opened the menu. Perhaps a bug with the `overlay`, I will double check why it does not do this is this particular case.

Reply:
Currently, I have to handle re-focusing in `ef-card` if this is an bug I will remove that code, Can you please check re-focus when close the menu by pressing key `esc` as well because re-focusing does not work the same.

Reply:
It is a bug, I have created PR: https://github.com/Refinitiv/refinitiv-ui/pull/306

Reply:
OK, I will remove my code for re-focusing

Reply:
I have already removed